src/easydiffraction/experiments/components/peak.py

```diff
@@ -379,25 +398,28 @@ def __init__(self):
 
 # --- Peak factory ---
 class PeakFactory:
+    ST = ScatteringTypeEnum
+    BM = BeamModeEnum
+    PPT = PeakProfileTypeEnum
     _supported = {
-        ScatteringTypeEnum.BRAGG: {
-            BeamModeEnum.CONSTANT_WAVELENGTH: {
-                PeakProfileTypeEnum.PSEUDO_VOIGT: ConstantWavelengthPseudoVoigt,
-                PeakProfileTypeEnum.SPLIT_PSEUDO_VOIGT: ConstantWavelengthSplitPseudoVoigt,
-                PeakProfileTypeEnum.THOMPSON_COX_HASTINGS: ConstantWavelengthThompsonCoxHastings,
+        ST.BRAGG: {
+            BM.CONSTANT_WAVELENGTH: {
+                PPT.PSEUDO_VOIGT: ConstantWavelengthPseudoVoigt,
+                PPT.SPLIT_PSEUDO_VOIGT: ConstantWavelengthSplitPseudoVoigt,
+                PPT.THOMPSON_COX_HASTINGS: ConstantWavelengthThompsonCoxHastings,
             },
-            BeamModeEnum.TIME_OF_FLIGHT: {
-                PeakProfileTypeEnum.PSEUDO_VOIGT: TimeOfFlightPseudoVoigt,
-                PeakProfileTypeEnum.PSEUDO_VOIGT_IKEDA_CARPENTER: TimeOfFlightPseudoVoigtIkedaCarpenter,
-                PeakProfileTypeEnum.PSEUDO_VOIGT_BACK_TO_BACK: TimeOfFlightPseudoVoigtBackToBackExponential,
+            BM.TIME_OF_FLIGHT: {
+                PPT.PSEUDO_VOIGT: TimeOfFlightPseudoVoigt,
+                PPT.PSEUDO_VOIGT_IKEDA_CARPENTER: TimeOfFlightPseudoVoigtIkedaCarpenter,
+                PPT.PSEUDO_VOIGT_BACK_TO_BACK: TimeOfFlightPseudoVoigtBackToBack,
             },
         },
-        ScatteringTypeEnum.TOTAL: {
-            BeamModeEnum.CONSTANT_WAVELENGTH: {
-                PeakProfileTypeEnum.GAUSSIAN_DAMPED_SINC: PairDistributionFunctionGaussianDampedSinc,
+        ST.TOTAL: {
+            BM.CONSTANT_WAVELENGTH: {
+                PPT.GAUSSIAN_DAMPED_SINC: PairDistrFuncGaussianDampedSinc,
```

Comment:
Again, you are renaming descriptive `PairDistributionFunctionGaussianDampedSinc` to shorter but much less meaningful `PairDistrFuncGaussianDampedSinc`. If you want to save space but be descriptive, why not `PDFGaussianDampedSinc`?

Reply:
Reverted back to full name